code/modules/mob/living/simple_animal/abnormality/zayin/oceanwaves.dm

```diff
@@ -168,121 +162,123 @@
 		L.reagents?.add_reagent_list(current_reagents)
 	return ..()
 
-// Soda cans
+// Good Soders
```

Comment:
Love what you're doing, but the mystery is what makes Ocean waves mechanic actually work.
If you knew you were drinking something good every time you drank it, without a clerk processing it, then that makes Oceanic Waves significantly more powerful.

It's already picked as much as Wellcheers is

Reply:
I will find a way to nerf it a bit. I don't like the current implementation of this abnormality.